mage_ai/api/resources/PipelineRunResource.py

```diff
@@ -199,6 +239,10 @@ async def process_collection(self, query_arg, meta, user, **kwargs):
             'next': has_next,
         }
 
+        if include_pipeline_uuids:
```

Comment:
qq: what's the purpose of returning all pipeline uuids?

Reply:
I need it to display to users so they can filter the project-wide pipeline runs by multiple pipelines, which actually seems useful (e.g. user just wants to see the pipeline runs for 2 or 3 pipelines).

Reply:
![image](https://github.com/mage-ai/mage-ai/assets/78053898/9ba2fd7e-f133-4780-8208-6ec1df0070c9)